src/main/java/io/github/jhipster/registry/config/ThymeleafConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package io.github.jhipster.registry.config;
```

Comment:
Better than JSP ? :o
Ok, it's deleted and also : 
- "thymeleaf.mode: XHTML" in all application*.yml
- the line "<logger name="org.thymeleaf" level="WARN"/>" in logback-spring.xml